### PR TITLE
feat(developer_manual): 28 files-to-vue upgrade guide

### PR DESCRIPTION
**Pull request by skjnldsv**
Some updates for files to vue:
- Removed APIs
- Added APIs
- Documented node events

### CODE REVIEW

**Comment by ChristophWurst** [comment]
screenshot pls

**Comment by skjnldsv** [comment]
> screenshot pls

image
image

File: developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_28.rst [hunk 1]
```diff
@@ -22,7 +22,17 @@ Front-end changes
...
+  https://nextcloud-libraries.github.io/nextcloud-files/functions/registerFileAction.html
+* New file menu: to register entries in the new file menu, please use the dedicated API from https://npmjs.org/@nextcloud/files or 
+  https://nextcloud-libraries.github.io/nextcloud-files/functions/addNewFileMenuEntry.html
+* Reminder from 27, to interact with the Files app router, use ``OCP.Files.Router``. See :ref:`FilesAPI`
+* To Interact with the Files app data, please use the following events. All of them have a `Node object <https://nextcloud-libraries.github.io/nextcloud-files/classes/Node.html>`_ as main parameter.
+
+  * ``files:node:created``: the node have been created
+  * ``files:node:deleted``: the node have been deleted
+  * ``files:node:moved``: the node have been moved (and its data is already updated)
+  * ``files:node:updated``: the node data have been updated
```
Comment (nickvergessen) [inline developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_28.rst:35]:
```suggestion
  * ``files:node:created``: the node has been created
  * ``files:node:deleted``: the node has been deleted
  * ``files:node:moved``: the node has been moved (and its data is already updated)
  * ``files:node:updated``: the node data has been updated
```

File: developer_manual/client_apis/files.rst [hunk 2]
```diff
@@ -14,8 +14,33 @@ We will update and document the new files app APIs and methods here.
...
+Events
+^^^^^^
+
+To listen to files or folder changes or to trigger changes, you need to use
+the ``event-bus`` and use the following events:
+
+  * ``files:node:created``: the node have been created
+  * ``files:node:deleted``: the node have been deleted
+  * ``files:node:moved``: the node have been moved (and its data is already updated)
+  * ``files:node:updated``: the node data have been updated
```
Comment (nickvergessen) [inline developer_manual/client_apis/files.rst:31]:
```suggestion
  * ``files:node:created``: the node has been created
  * ``files:node:deleted``: the node has been deleted
  * ``files:node:moved``: the node has been moved (and its data is already updated)
  * ``files:node:updated``: the node data has been updated
```

**Review by nickvergessen — APPROVED** [review]
Good otherwise

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**